### DISCUSSION TITLE
[Idea] Configure GLPI-AGENT directly from MDM

### DISCUSSION THREAD

**Discussion by ASLLR** [glpi-project/glpi-agent#456] [Ideas]
### Your idea

Hi everyone, 

I need to deploy the android GLPI agent on a number of terminals (more than 1000). I can see with the version of october 2021 we can générate a QRcode to automatically push the configuration (with subscription), but it requires having the terminal in hand. 

Do you have in the roadmap of the android version, the possibility to upload directly the configuration from the MDM like, for example, The DAVx5 managed app ?

Thank in advance for reply. 

Have à nice day

**Comment by g-bougard** [comment]
Hi @ASLLR 

I'm not involved in the development of android glpi-agent for which the public project is indeed there: https://github.com/glpi-project/android-inventory-agent

So I can't answer you. But I think @stonebuzz should be able to answer you after he'll come back from his holidays.

**Comment by ASLLR** [comment]
Hello @stonebuzz, are you back from your holidays? I hope they were good :) When you have the time, please, give me feedback on this request. 

Have a nice day, 
Best regards,

**Comment by stonebuzz** [comment]
Hi @ASLLR 

It is indeed possible to enhance the agent to be configured via an MDM

https://developer.android.com/work/managed-configurations

Several things to note:

- This is not a small feature (ideally funding should be appreciate)
- We don't have an MDM for testing

Best regards

**Reply by ASLLR** [reply]
Hi @stonebuzz i'm really sorry to answer you after 1 year ... but, in this year we're bought a glpi network licence with ticket to Teclib. I hope this can help with development, this is part of our contribution.

This feature is always needed for us, do you have any other requests ? what would be the cost of such a development ? This feature is in road map ? We work with Infotel and we could help for this if needed. 

Thank you in advance,

**Reply by stonebuzz** [reply]
Great news, now you can contact us from your partner

Best regards
